### PR TITLE
Reagent containers liquid sloshing

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request

https://github.com/user-attachments/assets/31605046-0b95-49ca-be62-a2baa517436d

https://github.com/user-attachments/assets/4f870d3f-e2ce-4414-a8ce-428d8172a516
## Why It's Good For The Game
feedback for if there is a liquid inside, especially for fire extinguishers
## Changelog
:cl: grungussuss
sound: reagent containers now make sounds if there is liquid in them when picked up or dropped
sound: fire extinguishers now make sounds when dropped or picked up
/:cl:

### CODE REVIEW

File: code/game/objects/items/extinguisher.dm [hunk 1]
```diff
@@ -66,6 +71,19 @@
 	context[SCREENTIP_CONTEXT_ALT_LMB] = "Empty"
 	return CONTEXTUAL_SCREENTIP_SET
 
+/obj/item/extinguisher/dropped(mob/user, silent)
+	. = ..()
+	var/obj/item/extinguisher/our_extinguisher = src
```
Comment (Ben10Omintrix) [inline code/game/objects/items/extinguisher.dm:76]: u dont need this. just access its vars directly

File: code/game/objects/items/extinguisher.dm [hunk 2]
```diff
@@ -66,6 +71,19 @@
...
+	if(fire_extinguisher_reagent_sloshing_sound && our_extinguisher.reagents.total_volume > 0)
+		playsound(src, fire_extinguisher_reagent_sloshing_sound, LIQUID_SLOSHING_SOUND_VOLUME, vary = TRUE, ignore_walls = FALSE)
+
+/obj/item/extinguisher/equipped(mob/user, slot, initial = FALSE)
+	. = ..()
+	var/obj/item/extinguisher/our_extinguisher = src
```
Comment (Ben10Omintrix) [inline code/game/objects/items/extinguisher.dm:82]: same as above

File: code/game/objects/items/extinguisher.dm [hunk 3]
```diff
@@ -66,6 +71,17 @@
 	context[SCREENTIP_CONTEXT_ALT_LMB] = "Empty"
 	return CONTEXTUAL_SCREENTIP_SET
 
+/obj/item/extinguisher/dropped(mob/user, silent)
+	. = ..()
+	if(fire_extinguisher_reagent_sloshing_sound && src.reagents.total_volume > 0)
```
Comment (Ben10Omintrix) [inline code/game/objects/items/extinguisher.dm:76]:
```suggestion
	if(fire_extinguisher_reagent_sloshing_sound && reagents.total_volume > 0)
```last thing i promise (also remove the rest of the srcs)

Comment (Ben10Omintrix) [inline code/game/objects/items/extinguisher.dm:76]:
```suggestion
	if(fire_extinguisher_reagent_sloshing_sound && reagents.total_volume > 0)
```

Comment (Ben10Omintrix) [reply code/game/objects/items/extinguisher.dm:76]: last thing i promise (probably). also dont need the other srcs

**Review by Ghommie — APPROVED** [review]

**Pull request merged**